compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -1106,6 +1106,9 @@ rustc_index::newtype_index! {
     /// are edges that go from a multi-successor node to a multi-predecessor node. This pass is
     /// needed because some analyses require that there are no critical edges in the CFG.
     ///
+    /// Note that this type is just an index into [`Body.basic_blocks`](Body::basic_blocks);
```

Comment:
Are there other places where `BasicBlock` is an index?

Reply:
I don't really understand the question?

Reply:
I wasn’t sure if there are other places where it is an index into a vector of blocks and I want to make sure I don’t mislead people by saying that it’s an index into specifically Body when in fact there are other places that hold a vector of blocks that are indexed by BasicBlock. Hope that was clearer!

Reply:
Ah I see, thanks for clarifying.

I don't think there are other lists this is an index into, but you can audit all `IndexVec<BasicBlock, _>` if you want to be sure.